src/libs/Navigation/Navigation.ts

```diff
@@ -196,7 +182,7 @@ function goBack(fallbackRoute, shouldEnforceFallback = false, shouldPopToTop = f
         return;
     }
 
-    // Add posibility to go back more than one screen in root navigator if that screen is on the stack.
+    // Add possibility to go back more than one screen in root navigator if that screen is on the stack.
     if (isCentralPaneFocused && fallbackRoute && distanceFromPathInRootNavigator > 0) {
         navigationRef.current.dispatch(StackActions.pop(distanceFromPathInRootNavigator));
```

Comment:
```suggestion
        navigationRef.current?.dispatch(StackActions.pop(distanceFromPathInRootNavigator));
```

Why `navigationRef.current` doesn't complain here? Same for line 191, `navigationRef.current.goBack();`.

Reply:
Surprisingly typescript is clever enough to infer that after this line:
 ```ts
    if (!navigationRef.current?.canGoBack()) {
        Log.hmmm('[Navigation] Unable to go back');
        return;
    }

navigationRef.current // no longer null
```